public/src/account/edit-profile-view.html

```diff
@@ -286,7 +286,7 @@ <h2>Edit Profile</h2>
       }
 
       _onInput() {
-        let valid = this.$.name.value.trim() ? true : false;
+        let valid = $core.isValidEmail(this.$.email.value.trim()) && this.$.email.value.trim().length > 0 && this.$.name.value.trim().length > 0 ? true : false;
         this.$.btnSave.disabled = !valid;
```

Comment:
usually this is redundant check in javascript
if (string)  is same as if (string && string.length > 0)

Reply:
Yes.  I should develop that habit to make the code cleaner.